tests/unit/plugins/modules/remote_management/lenovoxcc/test_xcc_redfish_command.py

```diff
@@ -0,0 +1,653 @@
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import json
+from ansible_collections.community.general.tests.unit.compat import mock
+from ansible_collections.community.general.tests.unit.compat.mock import patch
+from ansible_collections.community.general.tests.unit.compat import unittest
+from ansible.module_utils import basic
+from ansible.module_utils._text import to_bytes
+import ansible_collections.community.general.plugins.modules.remote_management.lenovoxcc.xcc_redfish_command as module
+
+
+def set_module_args(args):
```

Comment:
Why do you inline this (and `AnsibleExitJson` and `AnsibleFailJson` etc.) instead of using them from ansible_collections.community.general.tests.unit.plugins.modules.utils?

Reply:
Thanks for reminder. I didn't realize that these functions have been defined already. I have updated the code.